src/Ticket.php

```diff
@@ -873,6 +875,18 @@ public function defineTabs($options = [])
             $this->addStandardTab('Log', $ong, $options);
         }
 
+        if (isset($PLUGIN_HOOKS["customtabs"]) && is_array($PLUGIN_HOOKS["customtabs"])) {
+            foreach ($PLUGIN_HOOKS['customtabs'] as $plugin => $hook_callback) {
+                if (!Plugin::isPluginActive($plugin) || !is_callable($hook_callback)) {
+                    continue;
+                }
+                $hook_result = $hook_callback(['item' => $ong]);
```

Comment:
Maybe it could be usefull to pass current item. Plugins may probably need to know what is the related item when adding/removing tabs.

```suggestion
                $hook_result = $hook_callback(['item' => $this, 'tabs' => $ong]);
```